### PR TITLE
Massive yamllint commit

### PR DESCRIPTION
**Pull request by kulinacs**
This will be broken up later, this commit is to gather opinions
on potential yamllint configs. The changes were:

* Truthy variables to true or false
* Adding '---' to the top of every yaml file
* Fixing indentation

DO NOT MERGE

### CODE REVIEW

File: tests/check.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# Should be run from root directory of project
+
+# Lint Playbooks
+find . -type f -name "*.yml" ! -path "*meta*" ! -path "*extra*" ! -path "*sg-*" | xargs yamllint
```
Comment (dinoocch) [inline tests/check.sh]: Can we just use a `yamllint` configuration file?
http://yamllint.readthedocs.io/en/latest/configuration.html#ignoring-paths

I appreciate the `find` though :smile_cat:

Comment (dinoocch) [reply tests/check.sh]: Also can we move this to a travis/other ci solution?

**Review by dinoocch — CHANGES_REQUESTED** [review]
:+1: :+1: 🎉 🥇 💯

File: .yamllint [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    max-end: 0
+  hyphens:
+    max-spaces-after: 1
+  indentation:
+    spaces: consistent
+    indent-sequences: yes
+    check-multi-line-strings: no
+  key-duplicates: enable
+  line-length:
+    max: 200
```
Comment (m-wynn) [inline .yamllint]: 200 seems like a lot.  I've seen a lot of modern projects go with 100, a few go with 120, and older projects shoot for 80.
100 is considered the length that allows for two columns of text on a larger monitor (i.e. `tput cols` is just over 200)
80 is considered the length that allows for two columns of text on a smaller/laptop monitor (i.e. `tput cols` is just over 160).  Obviously we don't need to maintain compatibility with old TTY's.

My vote is that we try 80.  If we run the linter and realize need to bump it to 100, we can do that.  If we need to go over 100, maybe we should rewrite that section entirely.

**Review by m-wynn — APPROVED** [review]
A lot of blame, but that's ok

File: .yamllint (hunk 2 above)
Comment (dinoocch) [reply .yamllint]: I like 120 :+1:

Comment (tdorianh) [reply .yamllint]: My preference is 100

**Review by tdorianh — APPROVED** [review]

**Pull request merged**